src/main/java/com/exasol/adapter/installer/dialect/Dialect.java

```diff
@@ -4,5 +4,5 @@
  * List of supported dialects to install.
  */
 public enum Dialect {
-    POSTGRESQL, MYSQL
+    POSTGRESQL, MYSQL, SQLSERVER, DB2
```

Comment:
That looks duplicate to me. You already have the information by the `DialectProfiles`

Reply:
This is a list of supported arguments for user input basically. Classes are there of course, but we need to find out somehow which dialect a user wants to install